MANIFEST

```diff
@@ -880,13 +880,24 @@ t/complex/xii.pdf
 t/complex/xii.tex
 t/complex/xii.xml
 t/daemon/amsarticle.tex
-t/daemon/broken.tex
 t/daemon/api/port.spec
 t/daemon/api/port.xml
 t/daemon/citations.tex
 t/daemon/complex/exhaustive.spec
 t/daemon/complex/exhaustive.xml
 t/daemon/document.tex
+t/daemon/fatal_100.tex
+t/daemon/fatal_egroup.tex
+t/daemon/fatal_misdefined.tex
+t/daemon/fatals/fatal_100.spec
+t/daemon/fatals/fatal_100.status
```

Comment:
Hadn't we removed `*.status` files?

Reply:
Dangers of time-traveling... removed again.